### PR TITLE
Convert app/web_hub_test.go t.Fatal calls into assert/require calls

### PR DESCRIPTION
**Pull request by niklabh**
#### Summary

Convert app/web_hub_test.go t.Fatal calls into assert/require calls
#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/12214

### CODE REVIEW

File: app/web_hub_test.go [hunk 1]
```diff
@@ -100,6 +100,6 @@ func TestHubStopRaceCondition(t *testing.T) {
 	select {
 	case <-done:
 	case <-time.After(15 * time.Second):
-		t.Fatalf("hub call did not return within 15 seconds after stop")
+		require.Falsef(t, true, "hub call did not return within 15 seconds after stop")
```
Comment (hanzei) [inline app/web_hub_test.go]: Isn't `require.False` enough here?

Comment (niklabh) [reply app/web_hub_test.go]: done

File: app/web_hub_test.go [hunk 2]
```diff
@@ -100,6 +100,6 @@ func TestHubStopRaceCondition(t *testing.T) {
 	select {
 	case <-done:
 	case <-time.After(15 * time.Second):
-		t.Fatalf("hub call did not return within 15 seconds after stop")
+		require.False(t, true, "hub call did not return within 15 seconds after stop")
```
Comment (hanzei) [inline app/web_hub_test.go]: I missed the whole point of `require.False(t,true)` in my first review. We can just use `require.FailNow` here.

Comment (niklabh) [reply app/web_hub_test.go]: done

**Comment by hanzei** [comment]
@imisshtml Would you please review this PR when you have the time?

**Comment by hanzei** [comment]
Thanks @niklabh :+1:

**Pull request merged**